robot/rospackages/src/mux_selector/scripts/mux_select_server.py

```diff
@@ -115,14 +115,10 @@ def select_device(device):
 def mux_select_server():
     global local
     local = False
-    if len(sys.argv) == 2:
-        if sys.argv[1] == "local":
+    for x in sys.argv:
+        if x == "local":
             print("Running in local mode")
             local = True
-        else:
```

Comment:
I don't think this error message should be removed. Makes it harder to debug when the time will come again.